app/src/main/java/org/mozilla/fenix/widget/SearchWidgetProvider.kt

```diff
@@ -20,8 +20,12 @@ import org.mozilla.fenix.HomeActivity
 import org.mozilla.fenix.IntentReceiverActivity
 import org.mozilla.fenix.R
 import org.mozilla.fenix.utils.Settings
+import android.os.Build
+import androidx.appcompat.widget.AppCompatDrawableManager
+import androidx.core.graphics.drawable.toBitmap
 
-class SearchWidgetProvider : AppWidgetProvider() {
+@Suppress("TooManyFunctions")
+    class SearchWidgetProvider : AppWidgetProvider() {
```

Comment:
Nit: whitespace.

Reply:
Thanks for that! I haven't noticed it. I am submitting a PR tomorrow for another issue and it also has small changes in this file. I have fixed the whitespace issue there.

Reply:
First thing tomorrow morning!

Reply:
Done!

Reply:
Thank you so much!